conans/client/profile_loader.py

```diff
@@ -62,6 +63,19 @@ def update_vars(self, included_vars):
         included_vars.update(self.vars)
         self.vars = included_vars
 
+    def _concat_vars(self):
+        tmp_vars = OrderedDict()
+        for key, value in self.vars.items():
+            if key[-1] == "+":  # Append syntax
+                key = key[:-1]
+                if key in tmp_vars:
+                    tmp_vars[key] += " " + value
```

Comment:
This works for an `OrderedDict`, but it doesn't for a regular `dict` (the `key` doesn't already exist). I'm not sure if this is documented behavior, but I'd try to avoid it, better to call `setdefault` and do not add the empty blank if the key doesn't already exist.

Reply:
It is checking ``if key in tmp_vars:``, so not an issue.